scripts/install

```diff
@@ -71,14 +73,26 @@ ynh_exec_warn_less yunohost firewall allow --no-upnp UDP $port_wg
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing dependencies..." --time --weight=1
+ynh_script_progression --message="Installing dependencies..." --weight=7
 
-ynh_install_extra_app_dependencies --repo="http://deb.debian.org/debian buster-backports main" --package="$pkg_dependencies"
+# Add buster-backports repo
+ynh_add_repo --uri="http://deb.debian.org/debian" --suite="buster-backports" --component="main" --name="$app"
+
+# Add pin-priority for wireguard packages
+ynh_pin_repo --package="wireguard*" --pin="origin deb http://deb.debian.org/debian buster-backports main" --priority=995 --name="$app"
+
+# Update the list of package with the new repo
+ynh_package_update
+
+ynh_add_app_dependencies --package="$pkg_dependencies"
+
+# Remove buster-backports repo and pin-priority
+ynh_remove_extra_repo --name=$app
```

Comment:
So from what I understand here the point is to pin only `wireguard*` instead of `*` (which is what the helper does). Would it make sense to add an option to the ynh_add_repo helper ? 4.1 is released and we're releasing misc micro-patches everyday these days so we could add this quickly to the 4.1.x ... idk if that makes sense

Reply:
You could, but it need to modify multiple helpers, if you think it could be done, We will return to the use of ynh_install_extra_app_dependencies

Reply:
We could indeed improve the helper, but it would delay further this PR. I guess we can live with the warning for the time being?

Reply:
>I guess we can live with the warning for the time being?

Just for precision, there is no warning emitted here, from what I remember

Reply:
Anyway agreed, let's live with this for now